kolibri/plugins/device_management/assets/src/views/select-content-page/index.vue

```diff
@@ -206,6 +208,15 @@
         transferChannelContent,
         waitForTaskToComplete,
         updateTreeViewTopic,
+        updateResourceCounts(store) {
```

Comment:
I think there should only be one channel that gets updated, so you could probably simplify this with using `Array.prototype.find`

```
const updatedChannel = availableChannels.find(channel => channel.id === transferredChannel.id)
store.dispatch('SET_TRANSFERRED_CHANNEL', updatedChannel);
```

See if something like that works.

Reply:
I updated it @jonboiser and it worked.